witch_doctor.gemspec

```diff
@@ -18,14 +18,12 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
   s.test_files = Dir["spec/**/*"]
 
-  s.add_dependency "rails", "~> 3.2"
+  s.add_development_dependency "rails", "~> 4.0.0"
```

Comment:
This is the main line I'm really not sure about...

Maybe it should be something like:

    s.add_dependency "rails", ">= 3.2"
    s.add_development_dependency "rails", "~> 4.0.0"

?